src/main/java/read/write/file/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package read.write.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Error reading file", e);
```

Comment:
The exception message should be more informative. Include the file path to indicate which file could not be read. For example: "Can't read from file at path: " + filePath